src/client-side-metrics/exporter.ts

```diff
@@ -328,7 +342,8 @@ export class CloudMonitoringExporter extends MetricExporter {
             maxRetryDelayMillis: 50000,
           },
         );
-        await this.monitoringClient.createTimeSeries(
+        const monitoringClient = new MetricServiceClient({projectId});
```

Comment:
we may need to authenticate for this project Id

Since we have a bug tracking this, we can probably address this later